tests/FSharp.Core.UnitTests/FSharp.Core/Microsoft.FSharp.Control/AsyncModule.fs

```diff
@@ -148,6 +148,7 @@ module LeakUtils =
 
 // ---------------------------------------------------
 
+// [<Collection(nameof FSharp.Test.DoNotRunInParallel)>]
 type AsyncModule() =
```

Comment:
Overall, this looks just like properly rewritten tests, so probably can go separately :)